src/math/p_div.c

```diff
@@ -29,3 +29,33 @@ void p_div_f32(const float *a, const float *b, float *c,
         *(c + i) = *(a + i) / *(b + i);
     }
 }
+
+/**
+ *
+ * Element wise vector division between input vector 'a' and scalar 'b'
+ *
+ * @param a     Pointer to input vector
+ *
+ * @param b     Pointer to input scalar
+ *
+ * @param c     Pointer to output vector
+ *
+ * @param n     Size of 'a' and 'c' vector.
+ *
+ * @param p     Number of processor to use (task parallelism)
+ *
+ * @param team  Team to work with 
+ *
+ * @return      None
+ *
+ */
+
+void p_div_s(const float *a, const float *b, float *c,
+               int n, int p, p_team_t team)
+{
+
+    int i;
+    for (i = 0; i < n; i++) {
+        *(c + i) = *(a + i) / *(b);
```

Comment:
Would be faster to invert *b and then multiply in the loop.  A compiler might figure that out for you but it's easy to do in the code so the complier doesn't have to.